examples/todo-list/src/main.rs

```diff
@@ -45,22 +51,30 @@ async fn add_todo(mut request: Request) -> Result<Response, Error> {
     let todo_form = TodoForm::from_request(&mut request).await.unwrap();
 
     {
-        let mut todos = TODOS.write().await;
-        todos.push(TodoItem {
+        let db = DB.get().unwrap();
+        TodoItem {
+            id: 0,
             title: todo_form.title,
-        });
+        }
+        .save(db)
+        .await
+        .unwrap();
     }
 
     Ok(reverse!(request, "index"))
 }
 
 async fn remove_todo(request: Request) -> Result<Response, Error> {
     let todo_id = request.path_param("todo_id").expect("todo_id not found");
-    let todo_id = todo_id.parse::<usize>().expect("todo_id is not a number");
+    let todo_id = todo_id.parse::<i32>().expect("todo_id is not a number");
 
     {
-        let mut todos = TODOS.write().await;
-        todos.remove(todo_id);
+        let db = DB.get().unwrap();
+        TodoItem::objects()
+            .filter(<TodoItem as Model>::Fields::ID.eq(todo_id))
```

Comment:
Eventually we'll have some nice macro that will turn the above into something like:

```rust
query!(TodoItem, id == todo_id)
```
(or something similar)

but I wanted to keep this PR smaller (and honestly, the current API isn't terribly bad either).